### PR TITLE
empty out contribs in prep of community repo

### PR DESCRIPTION
**Pull request by apanicker-nflx**
Pull Request type
----

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [x] Other (please describe): 
Moving classes out of contribs module in preparation of conductor-community repo

Changes in this PR
----
Introducing the new repo - https://github.com/Netflix/conductor-community

New Modules:
- `awss3-storage`
-  `awssqs-event-queue` 
-  `json-jq-task`
- `http-task`

Removed Modules (moved to conductor-community):
- `mysql-persistence`
- `postgres-persistence`
- `postgres-external-storage`
- `azurebloblob-storage`
- `es7-persistence`

Changes to `contribs` module:
- `conductor-contribs` is no longer being published, each underlying implementation will be published as a separate dependency
- `HTTP` system task is moved to separate module
- `LocalOnlyLock` lock implementation is moved to core module
- `S3 Storage` is moved to separate module
- All other implementations within this module are moved to conductor-community

### CODE REVIEW

**Review by jxu-nflx — APPROVED** [review]

File: contribs/src/main/resources/META-INF/additional-spring-configuration-metadata.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
       "name": "conductor.event-queues.sqs.enabled",
       "type": "java.lang.Boolean",
       "description": "Enable the use of AWS SQS implementation to provide queues for consuming events.",
-      "sourceType": "com.netflix.conductor.contribs.queue.sqs.config.SQSEventQueueConfiguration"
+      "sourceType": "com.netflix.conductor.sqs.config.SQSEventQueueConfiguration"
```
Comment (aravindanr) [inline contribs/src/main/resources/META-INF/additional-spring-configuration-metadata.json:53]: Looks like the contribs module is being removed, but this file remains. Should this file go the new awssqs-queue module?

**Comment by aravindanr** [comment]
Should `HTTP` task be in its own module, like `jsonjq-task`? Adding `HTTP` to core bloats it with "web" related dependencies.

File: contribs/src/main/resources/META-INF/additional-spring-configuration-metadata.json (hunk 1 above)
Comment (apanicker-nflx) [reply contribs/src/main/resources/META-INF/additional-spring-configuration-metadata.json:53]: ah yes, this fell off my to-do list, fixing this now

**Comment by apanicker-nflx** [comment]
> Should `HTTP` task be in its own module, like `jsonjq-task`? Adding `HTTP` to core bloats it with "web" related dependencies.

Good point, will do that.

File: core/build.gradle [hunk 2]
```diff
@@ -25,6 +25,8 @@ dependencies {
 
     implementation "com.google.protobuf:protobuf-java:${revProtoBuf}"
 
+    implementation "javax.ws.rs:jsr311-api:${revJsr311Api}"
```
Comment (aravindanr) [inline core/build.gradle:28]: now that HTTP is in its own module, is this dependency still needed?

Comment (apanicker-nflx) [reply core/build.gradle]: fixed

**Pull request merged**